VirtoCommerce.MarketingModule.Web/Scripts/promotion/blades/catalog-entry-list.js

```diff
@@ -10,6 +10,7 @@ angular.module('virtoCommerce.marketingModule')
             $scope.saveChanges = function () {
                 this.blade.promotion.productIds = this.blade.productIds;
                 this.blade.promotion.productNames = _.pluck(selectedListEntries, 'name');
+                this.blade.parentBlade.$scope.$$childTail.productCodes[selectedListEntries[0].id] = selectedListEntries[0].code;
```

Comment:
Do we really need to use $$childTail here?

Reply:
After my researching how to work blades and them scopes, I'm found only this way. But I can mistake, of course.